ChangeLog

```diff
@@ -15,7 +15,9 @@
 === Miscellaneous ===
 
 === Bug fixes ===
-
+ * #1124 (DistributionFactory::buildAsXXX methods not documented)
+ * #1443 (Not all distributions have a getRoughness() method)
+ * #1485 (The Normal().getRoughness() method is wrong)
```

Comment:
this one should go to the 1.15 ; can you move it to target the 1.15 branch, else you can edit the commit and i'll cherry-pick to my misc fixes pr that targets 1.15

Reply:
Done